notes.md

```diff
@@ -101,19 +101,19 @@ The position in some Elements refers to the position, in octets, from the beginn
 
 #### Soft linking
 
-Soft linking is used by codec chapters. They can reference another Segment and jump on that Segment. The way the Segments are described are internal to the chapter codec and unknown to the Matroska level. But there are Elements in the Segment Information (such as `ChapterTranslate`) that can translate a value representing a Segment in the chapter codec and to the current Segment UID. All Segments that could be used in a file/segment this way should be marked as members of the same family (SegmentFamily), so that the player can quickly switch from one to the other.
+Soft Linking is used by codec chapters. They can reference another Segment and jump on that Segment. The way the Segments are described are internal to the chapter codec and unknown to the Matroska level. But there are Elements in the Segment Information (such as `ChapterTranslate`) that can translate a value representing a Segment in the chapter codec and to the current Segment UID. All Segments that could be used in a Linked Segment in this way SHOULD be marked as members of the same family via the SegmentFamily Element, so that the player can quickly switch from one to the other.
 
 #### Medium linking
 
-This kind of linking is a mix between hard and soft linking. Each Segment linked is independant from the other (standalone, unlike hard linked ones). But it should be treated as a hard-link by the player. Medium linking is done through chapters using the `ChapterSegmentUID` Element and only makes sense for ordered editions. The Segment matching the UID should be played as if it was part of the original Segment (Segment it's linked from) and then resume playback in the original Segment. That means the timecodes of the following content should be shifted by the duration of the linked Segment. As for hard-linking, the resulting Segment edition should be played and considered as one.
+This kind of linking is a mix between Hard Linking and Soft Linking. Each Segment linked is independant from the others (standalone, unlike hard linked ones). But it should be treated as a hard-link by the player. Medium linking is done through chapters using the `ChapterSegmentUID` Element and only makes sense for ordered editions. The Segment matching the UID should be played as if it was part of the original Segment (Segment it's linked from) and then resume playback in the original Segment. That means the timecodes of the following content should be shifted by the duration of the linked Segment. As for hard-linking, the resulting Segment edition should be played and considered as one.
```

Comment:
- `…unlike Hard Linked ones…` and similar issues in other places
- And why is `Medium linking` not capitalized like `Hard linking`?
- `…the UID SHOULD be played…`?
- `…following content SHOULD be shifted…`?